backend/corpora/common/corpora_orm.py

```diff
@@ -12,20 +13,57 @@
     ForeignKeyConstraint,
     Integer,
     String,
-    text,
 )
+from sqlalchemy.ext.declarative import DeclarativeMeta
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship, sessionmaker
 
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from common.corpora_config import CorporaDbConfig
+from .corpora_config import CorporaDbConfig
 
-Base = declarative_base()
-deployment_stage = os.environ["DEPLOYMENT_STAGE"]
 
-DEFAULT_DATETIME = text("now()")
+class TransformingBase(object):
+    """
+    Add functionality to transform a Base object, and recursively transform its linked entities.
+    """
+
+    relationship_to_dict = True  # Set to True if entities with a relationship to this should be converted to a dict.
+
+    def __iter__(self):
+        return self.to_dict().iteritems()
+
+    def to_dict(self, relation_to_dict: bool = None, backref: "Base" = None) -> dict:
+        """
+
+        :param relation_to_dict: If true, any connected relationships will be converted to dictionaries.
```

Comment:
Perhaps a more descriptive variable name? `convert_child_relationships_to_dict`. Names are cheap so erring on the side of longer more descriptive names is better.